android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -116,6 +121,22 @@ public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
             if (rnOptions.hasKey("sendDefaultPii")) {
                 options.setSendDefaultPii(rnOptions.getBoolean("sendDefaultPii"));
             }
+            if (rnOptions.hasKey("enableAutoPerformanceTracking")) {
+                if (rnOptions.getBoolean("enableAutoPerformanceTracking")) {
+                    if (RNSentryModule.frameMetricsAggregator == null) {
+                        RNSentryModule.frameMetricsAggregator = new FrameMetricsAggregator();
+                    }
+
+                    // Only add the current activity to frames metrics tracking if auto performance is on.
+                    Activity currentActivity = getCurrentActivity();
+
+                    if (currentActivity != null) {
+                        RNSentryModule.frameMetricsAggregator.add(currentActivity);
+                    }
+                } else {
+                    this.disableNativeFramesTracking();
```

Comment:
Shouldnt this also run if `rnOptions.hasKey("enableAutoPerformanceTracking"))` is false?